packages/babel-parser/test/fixtures/flow/class-private-property/variance/input.js

```diff
@@ -0,0 +1,4 @@
+class A {
+  +#foo;
+  -#foo;
+}
```

Comment:
The current main incorrectly parse `+#foo` as `ClassProperty` instead of `ClassPrivateProperty`. This is now fixed by a new `parsePropertyNamePrefixOperator` hook for the Flow plugin.